loleaflet/src/control/Toolbar.js

```diff
@@ -659,5 +659,23 @@ L.Map.include({
 				}, 0);
 			}
 		});
-	}
+	},
+
+	openRevisionHistory: function () {
+		var map = this;
+		// if we are being loaded inside an iframe, ask
+		// our host to show revision history mode
+		map.fire('postMessage', {msgId: 'rev-history', args: {Deprecated: true}});
+		map.fire('postMessage', {msgId: 'UI_FileVersions'});
+	},
+	openShare: function () {
+		var map = this;
+		map.fire('postMessage', {msgId: 'shareas', args: {Deprecated: true}});
+		map.fire('postMessage', {msgId: 'UI_Share'});
+	},
+	openSaveAs: function () {
+		var map = this;
+		map.fire('postMessage', {msgId: 'saveas', args: {Deprecated: true}});
+		map.fire('postMessage', {msgId: 'UI_SaveAs'});
```

Comment:
here I see two lines

Reply:
because initially I was testing and if I had only `map.fire('postMessage', {msgId: 'UI_SaveAs'});` I was getting sometimes failed results locally using docker... but I re-test it again with new clean container and running make clean first and somehow I don't get that weird behaviour now, so I guess it was some weird issue non relate to this. I will be updating this thanks.